Blutalkohol.java

```diff
@@ -0,0 +1,76 @@
+package Blutalkohol;
+
+import java.util.Scanner;
+public class Blutalkohol {
+
+	public static double rundenZweiteNachkommastelle(double zahl)
+		
+		{
+			return Math.round(zahl * 100) / 100.0;
+		}
+		
+		public static void main(String[] args) 
+		{
+			System.out.println("***BLUTALKOHOL-RECHNER***");
+			
+			Scanner sc = new Scanner(System.in);
+			
+		//AUSWAHL Mann/Frau/Kind:
+			
+			// r: Verteilungsfaktor im Körper
+			System.out.println("Bist du ein Mann, eine Frau oder ein Kind?");
+			
+			double verteilFaktImKörper = 0;
+			
+			while (verteilFaktImKörper == 0) 
```

Comment:
doubles are not compared with == since floating point numbers have precision issues